docs/Intro/welcome.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to Hyperledger fabric
+
+The Linux Foundation Hyperledger Project(1) is the result of collaboration between project members(2) 
+who share the common goal of designing a business-ready, open-source standard for blockchain networks. 
+Extensive research, planning and implementation by this community has led to the Linux Foundation adopting 
+Hyperledger fabric as its open source standard for blockchain network architecture.
+
+The Hyperledger fabric blockchain standard is a base architecture, with built-in functionality, security 
+and platform independence for blockchain networks. The open standard facilitates adoption by any public or 
+private organization or enterprise, and in turn by their constituent members.
+
+References:  
+(1) Hyperledger Project: https://www.hyperledger.org/  
+(2) Hyperledger Project members: https://www.hyperledger.org/about/members
+
+## Audience
```

Comment:
So, basically everyone. This is far too broad. I think that ideally you would direct specific audiences to specific sections that have relevance to their interests.